### PR TITLE
Fix test for `recommended` rule doc notice

### PR DESCRIPTION
**Pull request by bmish**
There was a bug in this test so we mistakenly added the `recommended` rule notice to some rule docs that are not in the `recommended` config.

The test will now correctly verify that the `recommended` rules include this notice and non-recommended rules don't.

### CODE REVIEW

File: test/package.mjs [hunk 1]
```diff
@@ -236,7 +236,7 @@ test('Every rule has a doc with the appropriate content', t => {
 		// Decide which notices should be shown at the top of the doc.
 		const expectedNotices = [];
 		const unexpectedNotices = [];
-		if (index.configs.recommended.rules[`unicorn/${ruleName}`]) {
+		if (index.configs.recommended.rules[`unicorn/${ruleName}`] === 'error') {
 			expectedNotices.push('configRecommended');
```
Comment (fisker) [inline test/package.mjs:240]: `!= 'off'`，, I believe `template-indent` was configured as `warn`.

Comment (bmish) [reply test/package.mjs:240]: Thanks, fixed

**Comment by bmish** [comment]
This is ready to be merged.

File: test/package.mjs (hunk 1 above)
Comment (bmish) [reply test/package.mjs:240]: Had to switch to `includes` to avoid [no-negated-condition](https://eslint.org/docs/rules/no-negated-condition).

**Pull request merged**